### DISCUSSION TITLE
Ultrasonix SDK files

### DISCUSSION THREAD

**Discussion by ojs1** [PlusToolkit/PlusLib#1009] [General]
Hi, 

I was wondering if there was somewhere I could download v6.1 (or newer) of the Ultrasonix Exam SDK? I am using a Sonix RP ultrasound system but the exam software is v.3.1 so I don't think it will be compatible with any of the plus versions available. 

Alternatively, is there a way to use the PlusServerLauncher with the old exam software? 

Thank you!

**Comment by lassoan** [comment]
To obtain the SDK legally, you probably need to contact BK ultrasound. If you have a service contract (not likely, because the Sonix RP is so old) then BK should be able to provide you the files; but otherwise they may not bother. Alternatively, you may contact university research groups who used Ultrasonix systems to see if they can help.

However, in the long term you probably want to switch to an ultrasound system that is still supported by its manufacturer. Telemed, Interson, BK, Clarius all have ultrasound systems with a research interface that Plus supports. Telemed and Interson are inexpensive; BK is expensive but high quality, Clarius is wireless.

**Reply by ojs1** [reply]
Thank you! I will reach out to Bk.
